### PR TITLE
document existing Keycloak configuration

### PR DESCRIPTION
**Pull request by ostrya**
Signed-off-by: Kai Helbig <kai.helbig@tngtech.com>

### CODE REVIEW

File: example-frontend-react/README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Prerequisites
+## Pre-requisites
```
Comment (christian-ertl) [inline example-frontend-react/README.md:3]: I think the hyphen should be removed.

Comment (ostrya) [reply example-frontend-react/README.md]: Indeed. Fascinating ...

**Review by christian-ertl — APPROVED** [review]

File: example-frontend-react/README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
-Before you can run this web app, you need a running Keycloak service.
-For this, you can either start a real Keycloak server:
+Before you can run this web app, you need a running Keycloak service. For this, you can either start
+a real Keycloak server:
 
 ```bash
 ./gradlew example-integration-docker:realKeycloakComposeUp
 ```
 
+The server has a pre-configured user `user` with password `user` in realm `realm` and
```
[the remark this answers is not in the document]
Comment (ostrya) [reply example-frontend-react/README.md:12]:
```suggestion
The server has a preconfigured user `user` with password `user` in realm `realm` and
```

**Comment by ostrya** [comment]
Sorry, I forgot to push two commits.

**Pull request merged**